src/DssSpell.sol

```diff
@@ -67,6 +67,13 @@ contract DssSpellAction is DssAction {
     address constant MCD_CLIP_CALC_MATIC_A = 0xdF8C347B06a31c6ED11f8213C2366348BFea68dB;
     address constant PIP_MATIC             = 0x8874964279302e6d4e523Fb1789981C39a1034Ba;
 
+    address constant PAX                     = 0x8E870D67F660D95d5be530380D0eC0bd388289E1;
+    address constant MCD_JOIN_PSM_PAX_A      = 0x7bbd8cA5e413bCa521C2c80D8d1908616894Cf21;
```

Comment:
We should use the original code for the Join adapter.

The new version is not approved and breaks the event api.

Reply:
Yeah, I think we do need to update this, but we have Lindy Effect on the old code and should stick with that on such short notice.

Reply:
I agree here, we should use the same exact versions of contracts for this new join and PSM right now, like we do with the other collaterals.
Then take some time to do optimizations and general improvements and get a v2 for them. Migration can be done in a later spell for both USDC and PAX.

Reply:
Ok so as Sam mentioned certain changes are necessary because this is a 18 decimals token.
However this `join` has included some other changes. This situation isn't ideal as we are force to put some new code and it is very tempting to fix other stuff. However I think I'd still prefer we deploy an adapter which is exact the same to the other excepting the decimals par. Then we take the time to come with a newer one for both which can be updated in the future.

Reply:
What's the difference between this `AuthGemJoin` and the one we used for `Sai`?

https://etherscan.io/address/0xad37fd42185Ba63009177058208dd1be4b136e6b#code

We should use the 0.5.12 version if possible because it's been formally verified and reviewed.

Reply:
There is no adapter that has the code we need. This one has a msgSender being added to the parameters which is required by the psm. I can deploy the sai adapter with a passed in msg sender if that's what you guys want. It's the least amount of code change.

Reply:
Going to resolve this convo. We decided to use the new GemJoin code and launch at 50mm Dai DC to keep within the surplus buffer.